Bottom: e2d86c570cdcf4687fc5f27cd935d818f4a8da80
Top:    668ecc8bee8ae5c1474eb97f0be7fa692bddf4b0
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-12 10:23:46 +0800

tuntap: IFF_TX_RING

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 7a5687f..2428b9f 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1892,7 +1892,7 @@ static int tun_set_iff(struct net *net, struct file *file, struct ifreq *ifr)
 
 		tun = netdev_priv(dev);
 		tun->dev = dev;
-		tun->flags = flags;
+		tun->flags = flags | IFF_TX_RING;
 		tun->txflt.count = 0;
 		tun->vnet_hdr_sz = sizeof(struct virtio_net_hdr);
